osl/interface_linux.go

```diff
@@ -30,6 +31,86 @@ type nwIface struct {
 	sync.Mutex
 }
 
+func (i *nwIface) MarshalJSON() ([]byte, error) {
```

Comment:
Honestly, I was hoping that the daemon restart with running containers should help us store less states on the store and have an ability to reconstruct states from running instances.

But seeing more states going into the db worries me @ the design level.

I guess we have to think about a design that helps us move away from the need to save more states.